app.js

```diff
@@ -1,6 +1,9 @@
 const express = require('express');
 const cors = require('cors');
 
+const users = require('./routes/users');
+const routes = require('./routes/route');
```

Comment:
let's keep the singularity consistent with plural `routes.js` (same for the route group)